### PR TITLE
Reduce concurrent builds down to 30.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
We only need more for um experiments, it's just nice to have for others. But it breaks things.

Related: https://github.com/google/fuzzbench/issues/1648

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
TBR

**Comment by jonathanmetzman** [comment]
CC @oliverchang @Alan32Liu

**Review by oliverchang — COMMENTED** [review]
LGTM

File: experiment/build/builder.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 from common import logs
 
 from experiment.build import build_utils
-from experiment.run_experiment import DEFAULT_CONCURRENT_BUILDS
+from experiment import run_experiment
```
Comment (Alan32Liu) [inline experiment/build/builder.py:37]: Out of curiosity: Why is importing `run_experiment` preferred over `DEFAULT_CONCURRENT_BUILDS ` here, given we only use the latter?

**Review by Alan32Liu — COMMENTED** [review]
LGTM

File: experiment/build/builder.py (hunk 1 above)
Comment (jonathanmetzman) [reply experiment/build/builder.py:37]: Youre only supposed to import packages/modules, not classes, variables or functions: https://google.github.io/styleguide/pyguide.html#22-imports

Comment (Alan32Liu) [reply experiment/build/builder.py:37]: Thanks!

**Pull request merged**